raft/tracker/inflights.go

```diff
@@ -115,7 +128,7 @@ func (in *Inflights) FreeLE(to uint64) {
 
 // Full returns true if no more messages can be sent at the moment.
 func (in *Inflights) Full() bool {
-	return in.count == in.size
+	return in.count == in.size || (in.bytes >= in.maxBytes && in.maxBytes > 0)
```

Comment:
Minor improvement: if `maxBytes` isn't set, then the right operand `in.bytes >= in.maxBytes` will not be evaluated at all. Since it's a new field, and defaults to 0, so it's disabled (0) by default. 

```suggestion
	return in.count == in.size || (in.maxBytes > 0 && in.bytes >= in.maxBytes)
```